aidants_connect_habilitation/templates/aidants_connect_habilitation/referent-form-view.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'layouts/main.html' %}
+{% load ac_common form_extras static %}
+
+{% block title %}
+  Aidants Connect - Personnes impliquées dans la structure {{ organisation }}
+{% endblock title %}
+
+{% block content %}
+  <div class="fr-container">
+    {% include 'layouts/_js-unavailable-alert.html' %}
+
+    <h1 class="fr-mb-2v">Formulaire d’habilitation</h1>
```

Comment:
```suggestion
    <h1 class="fr-mb-2v">Formulaire d’habilitation</h1>
```
Pour garder une présentation cohérente avec les autres étapes